util/testutil/integration/run.go

```diff
@@ -423,3 +423,10 @@ func prepareValueMatrix(tc testConf) []matrixValue {
 	}
 	return m
 }
+
+// Skips tests on Windows
+func SkipOnWindows(t *testing.T) {
```

Comment:
Could we rename this to be `SkipOnPlatform`, and have it take a `...string`?

At some point in the future, I imagine we'd want to add BSD integration tests (since it's supported). There's quite a few tests that we skip that would probably apply to both.

Reply:
Makes sense, will do that.